tests/plugins/test_generate_maven_metadata.py

```diff
@@ -351,7 +462,7 @@ def test_generate_maven_metadata(tmpdir, docker_tasker, user_params):
 
     components = {(component['filename'], component['checksum'], component['checksum_type'])
                   for component in plugin_result.get('components')}
-    remote_source_files = results[GenerateMavenMetadataPlugin.key]['remote_source_files']
+    remote_source_files = plugin_result.get('remote_source_files')
```

Comment:
we should test here also pnc_build_metadata

Reply:
I've added a separate test for that.

Reply:
Added test for generate_maven_metadata as well